ert3_examples/snake_oil/experiments/doe/ensemble.yml

```diff
@@ -46,4 +30,4 @@ output:
 
 forward_model:
   driver: local
-  stage: run_simulator
+  stage: run_simulator_dm
```

Comment:
I think we should be bold in such a PR and replace the existing stage and thereby say that this is the way to do it by only providing one stage.

Reply:
Right, I thought that this could show different approaches but you might be right.